src/messages/MessageBuilder.cpp

```diff
@@ -343,16 +345,16 @@ MessageBuilder::MessageBuilder(const UnbanAction &action)
     this->message().flags.set(MessageFlag::System);
     this->message().flags.set(MessageFlag::Untimeout);
 
-    this->message().timeoutUser = action.target.name;
+    this->message().timeoutUser = action.target.login;
 
     QString text;
 
-    this->emplaceSystemTextAndUpdate(action.source.name, text)
-        ->setLink({Link::UserInfo, action.source.name});
+    this->emplaceSystemTextAndUpdate(action.target.login, text)
+        ->setLink({Link::UserInfo, action.target.login});
```

Comment:
You've replaced source with target here - is this intentional?

Reply:
Here, and in all other cases with the same comment I must've used bulk-replace in a wrong way, thanks for noticing!